### ISSUE TITLE
Checkpoints for LibriTTS experiments from paper?

### ISSUE THREAD

**Issue by lcflint** [FunAudioLLM/CosyVoice#405]
Hey there! Been testing this model out for a while now and have been loving the results.

Is there a chance that you would be able to release checkpoints for the LibriTTS experiments performed in the paper?

I've been looking to deploy the model on edge devices using English as its only language, but the RTF is unfortunately too high with the standard multi-lingual model (approx. 1.7-2.0). This is a shame since the quality of the model far outperforms anything else I've tested (FishSpeech, XTTSv2, OpenVoice, etc.)

It'd be really helpful for users if the checkpoints for the experiments (particularly Exp-3-LibriTTS from the paper) were released. It seems like the tiny model would be very performant while still having good enough accuracy, and it'd help the paper have even more reach and impact if users could access these results

Hope it'd be possible for you guys to release these, many thanks

**Comment by aluminumbox** [comment]
we will release a 25hz model maybe in this month, the rtf should reduce by half. I think checkpoints form libritts is not suitable for deployment because its training data is far not enough.

**Comment by lcflint** [comment]
That sounds good, thanks for the heads up
